screens/SettingsScreen.tsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { Button } from 'react-native';
```

Comment:
Can we use Button from `react-native-elements`?

Reply:
Yea, but I have planned to do it during adding other things here.

Reply:
Okay, so we can can keep it for now.